spec/unit/views/components/sidebar_section_spec.rb

```diff
@@ -44,4 +44,25 @@
     end
   end
 
+  context 'with attributes_table for resource' do
+    let(:post) { Post.create!(title: "Testing.") }
+    let(:section) do
+      ActiveAdmin::SidebarSection.new('Summary', options) do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.